src/hostnet/lib/host_uwt.ml

```diff
@@ -82,11 +82,14 @@ module Sockets = struct
         loop () in
       loop ()
 
-    let input ~reply ~src:(src, src_port) ~dst:(dst, dst_port) ~payload =
+    let input ?desc ~reply ~src:(src, src_port) ~dst:(dst, dst_port) ~payload () =
       (if Hashtbl.mem table (src, src_port, dst, dst_port) then begin
           Lwt.return (Some (Hashtbl.find table (src, src_port, dst, dst_port)))
         end else begin
-         let description = Ipaddr.to_string src ^ ":" ^ (string_of_int src_port) ^ "-" ^ (Ipaddr.to_string dst) ^ ":" ^ (string_of_int dst_port) in
+         let base_description = Ipaddr.to_string src ^ ":" ^ (string_of_int src_port) ^ "-" ^ (Ipaddr.to_string dst) ^ ":" ^ (string_of_int dst_port) in
```

Comment:
Spoke with @avsm and @yomimono IRL and we decided that refactoring the logging here could wait for another PR. I'll do the `String.concat` thing now though.